app/controllers/cip_change_alert_controller.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class CipChangeAlertController < ApplicationController
+  before_action :authenticate_user!
+
+  def start
+    @original_cip_name = cip_change_message.original_cip.name
+    @new_cip_name = cip_change_message.original_cip.name
```

Comment:
should this be `cip_change_message.new_name.name`??

Reply:
yes.. I originally had this bug but I fixed it when I changed these into helper methods below. So they don't need to be here any more 😅